studio/components/interfaces/Organization/BillingSettingsV2/Subscription/ExitSurveyModal.tsx

```diff
@@ -82,25 +85,26 @@ const ExitSurveyModal = ({ visible, onClose }: ExitSurveyModalProps) => {
     }
   }
 
-  const downgradeProject = async (values?: any) => {
+  const downgradeOrganization = async (values?: any) => {
     // Update the subscription first, followed by posting the exit survey if successful
     // If compute instance is present within the existing subscription, then a restart will be triggered
-    if (!projectRef) return console.error('Project ref is required')
+    if (!slug) return console.error('Slug is required')
 
     try {
-      await updateSubscriptionTier({ projectRef, tier: 'tier_free' })
+      await updateOrgSubscription({ slug, tier: 'tier_free' })
       resetCaptcha()
     } catch (error: any) {
       return ui.setNotification({
         error,
         category: 'error',
-        message: `Failed to cancel subscription: ${error.message}`,
+        message: `Failed to downgrade subscription: ${error.message}`,
       })
     }
 
     try {
+      // [Joshen TODO] Update to accept org slug once endpoint is updated
       const feedbackRes = await post(`${API_URL}/feedback/downgrade`, {
-        projectRef,
+        projectRef: '',
```

Comment:
Is this right?

Reply:
yup! this is just temp for now - we'll need to update the feedback endpoint to accept an org slug instead of project ref hence why